src/dotnet/APIView/APIViewWeb/Languages/PythonLanguageService.cs

```diff
@@ -19,6 +19,8 @@ public class PythonLanguageService : LanguageProcessor
         public override string Extension { get; } = ".whl";
         public override string VersionString { get; } = "0.3.2";
 
+        public override bool IsReviewGenByPipeline { get; } = false;
```

Comment:
Why do we override with the same value?

Reply:
I left this as a placeholder to flip. But forgot that I can just remove false when value was changed from true to false. :(

Reply:
Removed override.